### PR TITLE
Corrections

### PR DESCRIPTION
**Pull request by TimElectrum**

### CODE REVIEW

File: src/docs/devguide/hugo/hugoBuild.sh [hunk 1]
```diff
@@ -35,8 +35,8 @@ echo ''
 echo ''
 echo '  + Running hugo build'
 echo ''
-docker run --name "hugo" -v ${BASE_DIR}/target/devguide/hugo:/src -v ${BASE_DIR}/target/devguide/site:/output -e "HUGO_THEME=hugo-material-docs" -e "HUGO_BASEURL=https://electrumpayments.github.io/airtime-service-interface-docs/" jojomi/hugo
-#docker run --name "hugo" -v ${BASE_DIR}/target/devguide/hugo:/src -v ${BASE_DIR}/target/devguide/site:/output -e "HUGO_THEME=hugo-material-docs" -e "HUGO_BASEURL=/" jojomi/hugo
+#docker run --name "hugo" -v ${BASE_DIR}/target/devguide/hugo:/src -v ${BASE_DIR}/target/devguide/site:/output -e "HUGO_THEME=hugo-material-docs" -e "HUGO_BASEURL=https://electrumpayments.github.io/airtime-service-interface-docs/" jojomi/hugo
```
Comment (mordi34) [inline src/docs/devguide/hugo/hugoBuild.sh]: revert this?

File: src/main/java/io/electrum/airtime/api/model/ErrorDetail.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
+package io.electrum.airtime.api.model;
```
Comment (mordi34) [inline src/main/java/io/electrum/airtime/api/model/ErrorDetail.java]: Is this class going to be removed from the Base? It should be right?

File: src/docs/devguide/hugo/hugoBuild.sh (hunk 1 above)
Comment (mordi34) [inline src/docs/devguide/hugo/hugoBuild.sh]: revert this

File: src/main/java/io/electrum/airtime/api/model/ErrorDetail.java (hunk 2 above)
Comment (mordi34) [inline src/main/java/io/electrum/airtime/api/model/ErrorDetail.java]: is this already removed from the base? I think it should be right?

File: src/main/java/io/electrum/airtime/api/model/Product.java [hunk 3]
```diff
@@ -44,20 +44,31 @@ public void setProductId(String productId) {
...
+    * <p>
     **/
+   @Deprecated
    public Product amount(LedgerAmount amount) {
       this.amount = amount;
       return this;
    }
 
-   @ApiModelProperty(value = "If the product identified by the product code is not a fixed price product then the amount field indicates the value of the product referred to.")
+   @ApiModelProperty(value = "<p>Deprecated: This field is deprecated. If supplied it shall be accepted but the Amounts model should be used in preference to this field. If both this field and Amounts are used then Amounts shall take preference. This field may be removed in a future version of the interface without notice.</p><p>If the product identified by the product code is not a fixed price product then the amount field indicates the value of the product referred to.<p>")
```
Comment (mordi34) [inline src/main/java/io/electrum/airtime/api/model/Product.java]: has the new amounts not been added or am I just missing it?

Comment (mordi34) [inline src/main/java/io/electrum/airtime/api/model/Product.java]: Oh this is now in the Request class and not in product now?

File: src/docs/devguide/hugo/hugoBuild.sh (hunk 1 above)
Comment (TimElectrum) [inline src/docs/devguide/hugo/hugoBuild.sh]: good catch, i always do this!

**Comment by TimElectrum** [comment]
please also request release notes so i don't forget abt them

**Pull request merged**